roles/pulp_health_check/meta/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+galaxy_info:
+  author: Pulp Team
+  description: A role to verify Pulp healthy.
+  issue_tracker_url: https://pulp.plan.io/projects/pulp/issues/new
+  license: GPL-2.0-or-later
+  company: Red Hat
+  min_ansible_version: 2.8
+  platforms:
+    - name: Debian
+      versions:
+        - buster
+    - name: Fedora
+      versions:
+        - 30
+        - 31
+    - name: EL
+      versions:
+        - 7
+        - 8
+  galaxy_tags:
+    - pulp
+    - pulpcore
+dependencies:
+  - pulp_api
```

Comment:
We should discuss this part of the design. My original thinking was that it would be run against the webserver host, or from an entirely different host but connecting to the webserver.

Reply:
I believe we don't need it anymore, it was from a previous tentative that went wrong and I forgot to remove